classes/classes/Items/Mutations.as

```diff
@@ -1265,6 +1253,21 @@
 				player.skinDesc = "fur";
 				player.underBody.restore(); // Restore the underbody for now
 			}
+			// Hooves - Tail and not taur
+			if (player.lowerBody != LOWER_BODY_TYPE_HOOFED && !player.isTaur() && player.tailType == TAIL_TYPE_HORSE && changes < changeLimit && rand(5) == 0) {
+				changes++;
+				if (player.lowerBody == LOWER_BODY_TYPE_HUMAN) outputText("\n\nYou stagger as your feet change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
+				if (player.lowerBody == LOWER_BODY_TYPE_DOG) outputText("\n\nYou stagger as your paws change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
+				if (player.lowerBody == LOWER_BODY_TYPE_NAGA) outputText("\n\nYou collapse as your sinuous snake-tail tears in half, shifting into legs.  The pain is immense, particularly in your new feet as they curl inward and transform into hooves!", false);
+				//Catch-all
+				if (player.lowerBody > LOWER_BODY_TYPE_NAGA) outputText("\n\nYou stagger as your " + player.feet() + " change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
```

Comment:
Disclaimer, I just robbed this from Minotaur Blood. So I guess I'll have to go peek at that as well o3o